server/cluster.go

```diff
@@ -366,6 +391,16 @@ func (n *ClusterNode) proxyToMaster(msg *ClusterReq) error {
 	return err
 }
 
+// proxyToMaster forwards request from topic proxy to topic master.
+func (n *ClusterNode) proxyToMasterAsync(msg *ClusterReq) error {
```

Comment:
I don't see any calls to this method.
Can you double check please?

Reply:
Oops. Fixed.